libs/shared/util-services/src/lib/build-edit-service-payload.ts

```diff
@@ -133,7 +133,7 @@ function refactoContainer({ service: container, request = {} }: containerProps):
     memory: container.memory,
     max_running_instances: container.max_running_instances,
     min_running_instances: container.min_running_instances,
-    registry_id: container.registry_id || '',
+    registry_id: container.registry_id || container.registry?.id || '',
```

Comment:
I have a special case here, that's why I added `container.registry?.id`, I think Container detail return an `registry` object